spec/numberformat.html

```diff
@@ -77,18 +77,21 @@ <h1>
         1. Let _style_ be _numberFormat_.[[Style]].
         1. If _style_ is *"currency"*, then
           1. Let _currency_ be _numberFormat_.[[Currency]].
-          1. Let _cDigits_ be CurrencyDigits(_currency_).
+          1. Let _currencyPrecision_ be _numberFormat_.[[CurrencyPrecision]].
+          1. Let _cDigits_ be CurrencyDigits(_currency_, _currencyPrecision_).
+          1. Let _defaultRoundingIncrement_ be CurrencyRoundingIncrement(_currency_, _currencyPrecision_).
           1. Let _mnfdDefault_ be _cDigits_.
           1. Let _mxfdDefault_ be _cDigits_.
         1. Else,
+          1. Let _defaultRoundingIncrement_ be 1.
```

Comment:
Observation: not sure how the spec scopes variables but this `Let` is not in the same lexical scope as the code below that reads from it. If it is a problem, you could `Let` it be 1 further up and then `Set` it to something else when needed.